functions/frequency_of_characters.py

```diff
@@ -0,0 +1,29 @@
+def frequency_of_characters(string):
+    """function to count the number of times of each character and prints the output.
+    Args:
+       string(str): The first parameter.
+
+    Returns:
+       Dict: The return value, dictionary with key - character and value - count of character.
+
+    Example:
+       string='Example' |  [('E', 1), ('x', 1), ('a', 1), ('m', 1), ('p', 1), ('l', 1), ('e', 1)]
```

Comment:
* Fix alignment
* Accoridng to the current output change the example in docstring